index.html

```diff
@@ -6,12 +6,21 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
     <meta http-equiv="X-UA-Compatible" content="ie=edge"/>
     <title>CodersCamp2020 | Pokemon Fight</title>
-    <link rel="stylesheet" href=""/>
+    <link rel="stylesheet" href="styles/app.css">
 </head>
 
 <body>
 
 
+    <div id="loading-view">
+        <p>Please wait a bit, we are looking for the Poké Flute to wake up Snorlax to fight.</p>
+        <img id="zzz" src="assets/zzz.png" alt="zzz">
+        <img src="assets/snorlax.png" alt="sleeping snorlax">
+    </div>
```

Comment:
Shouldn't it be a div with the id "app" in which we will then load different templates?

Reply:
I thought about it. This is the start screen we see before our application loads (similar to our names in the pokemon quiz). 
I was thinking about create in html body two divs: first with class or id equal "content" or "app" to load templates and second "footer" with only discription as u see here. But i'm not sure where to put this loading view. In my opinion this current view shouldnt be a template.    
What do you think about this solution for html body:
<body>

    <div id="app">
        <div id="loading-view">
            <p>Please wait a bit, we are looking for the Poké Flute to wake up Snorlax to fight.</p>
            <img id="zzz" src="assets/zzz.png" alt="zzz">
            <img src="assets/snorlax.png" alt="sleeping snorlax">
        </div>
        <!-- below div for others views -->
        <template id="current-view"></template>
    </div>
    
    <footer>
        TypeScript Mangos project for CodersCamp2020
    </footer>

    <script src="dist/index.js"></script>

</body>

Reply:
```suggestion
<div id="app">
    <div id="loading-view">
        <p>Please wait a bit, we are looking for the Poké Flute to wake up Snorlax to fight.</p>
        <img id="zzz" src="assets/zzz.png" alt="zzz">
        <img src="assets/snorlax.png" alt="sleeping snorlax">
    </div>
</div>

<footer>
    TypeScript Mangos project for CodersCamp2020
</footer>

    <!-- below div for others views -->
    <template id="current-view"></template>

<script src="dist/index.js"></script>
```
I like your suggestion! We could also put the templates outside of the app div (like we did last time and like I suggested aboce), but I'm not sure if it would make any difference, so maybe it sould be up to you to decide :)

Reply:
I agree with @memeraki . Loading view won't be a template but we will use templates for other views.